services/catarse/lib/tasks/balance_transfer.rake

```diff
@@ -31,7 +31,7 @@ namespace :balance_transfer do
   desc 'update balance_transfers status'
   task update_status: :environment do
     PagarMe.api_key = CatarseSettings[:pagarme_api_key]
-    BalanceTransfer.processing.each do |bt|
+    BalanceTransfer.processing.where('transfer_id is not null').each do |bt|
```

Comment:
Sugestão:
Nossa versão do Rails já suporta o uso do `where.not` e também quando se tratar de iteração de registros do banco, é utilizar o `.find_each` pois caso haja muitos registros, ele não carregará todos de uma vez na memória.
```suggestion
    BalanceTransfer.processing.where.not(transfer_id: nil).find_each do |bt|
```

Reply:
estou aplicando aqui

o @devton tinha falado em colocar direto pro main, pois vai ser uma alteração só pra não dar esse problema caso a transferência tenha sido feita pela transfeera.

Reply:
@grpse o @stephannv tinha voltado o main, o ideal aqui seria colocar isso no mesmo PR ali das outras alterações.

Reply:
@grpse tu coloca essa modificação no branch da transfeera e faz rebase do branch com o main, pois tem coisas no rubocop que ele tá quebrando.